CHANGELOG.md

```diff
@@ -127,7 +127,25 @@ binary | sha1 hash | md5 hash
 
 ## Known Issues and Important Steps before Upgrading
 
-* *Instructions coming soon*
+The following versions of Docker Engine are supported - *v1.10*, *v1.11*
+Although *v1.9* is still compatible, we recommend upgrading to one of the supported versions.
+
+### kubernetes Core Known Issues
+
+- Kube Proxy crashes infrequently due to a docker bug ([#24000](https://github.com/docker/docker/issues/24000))
+- CORS works only in insecure mode ([24086](https://github.com/kubernetes/kubernetes/issues/24086))
+- Persistent volume claims gets added incorrectly after being deleted under stress. Happens very infrequently. ([#26082](https://github.com/kubernetes/kubernetes/issues/26082))
+ 
+### Docker runtime Known Issues
+
+- Kernel crash with Aufs storage driver on Debian Jessie (#[27885](https://github.com/kubernetes/kubernetes/issues/27885))
```

Comment:
https://github.com/kubernetes/node-problem-detector/pull/22 is not merged yet. Will update the document after the PR gets merged. >"<